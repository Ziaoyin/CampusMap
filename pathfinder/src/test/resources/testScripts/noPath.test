# tests if there is no given path, the output is correct
CreateGraph graph
AddNode graph n1
AddNode graph n2
FindPath graph n1 n2
FindPath graph n2 n1

AddNode graph n3
AddEdge graph n1 n2 3
FindPath graph n1 n3
FindPath graph n3 n1
FindPath graph n3 n2
FindPath graph n2 n3

AddNode graph n4
AddNode graph n5
AddEdge graph n4 n5 3
FindPath graph n1 n4
FindPath graph n2 n5
FindPath graph n3 n4