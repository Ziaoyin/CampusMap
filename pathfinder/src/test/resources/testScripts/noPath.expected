# tests if there is no given path, the output is correct
created graph graph
added node n1 to graph
added node n2 to graph
path from n1 to n2:
no path found
path from n2 to n1:
no path found

added node n3 to graph
added edge 3.000 from n1 to n2 in graph
path from n1 to n3:
no path found
path from n3 to n1:
no path found
path from n3 to n2:
no path found
path from n2 to n3:
no path found

added node n4 to graph
added node n5 to graph
added edge 3.000 from n4 to n5 in graph
path from n1 to n4:
no path found
path from n2 to n5:
no path found
path from n3 to n4:
no path found